os/se/ameba/rtl_security_api_wrapper_tz.c

```diff
@@ -66,6 +66,28 @@ static int ns_flash_write(uint32_t address, uint32_t len, uint8_t *data)
 	return ret;
 }
 
+/* Verify Flash Data Function */
+int rtl_ss_flash_read(uint32_t address, uint32_t len, uint8_t *data, int en_display)
+{
+	int ret;
+	flash_t flash;
+
+	ret = flash_stream_read(&flash, address, len, data);
+
+	if (en_display){
```

Comment:
This if condition works without checking the return of `flash_stream_read`, but the `data` is from it.
I think it is better to check the return of `flash_stream_read` first.

Reply:
Modified